manifest.xml

```diff
@@ -0,0 +1,13 @@
+<package>
+  <description brief="Driver library for u-blox GPS modules">
+  Driver library for u-blox GPS modules
+  </description>
+  <author>Gabriel Arjones/arjones@arjones.com</author>
+  <license></license>
```

Comment:
Set the license as BSD 2-clause

Also add a &lt;copyright&gt; tag with TideWise

Reply:
I added the 3-clause one because that's the one you used for the IMU driver. Which one should I add here? May we define a default?

Reply:
3-clause. Sorry. You're right.

We need to define a policy on the license stuff, I agree. So far, I'm making the drivers BSD.

Reply:
> I added the 3-clause one because that's the one you used for the IMU driver. Which one should I add here? May we define a default?

"Added" ? It's not listed here neither in the README. Where did you add it ?

(Actually, removing the boilerplate README and fill it with minimal info would be good too)

Reply:
> "Added" ? It's not listed here neither in the README. Where did you add it ?

It's on master, I selected it when I created the repository through github's UI.

> removing the boilerplate README and fill it with minimal info 

Sure.. I was kinda rushing because you mentioned you wanted to test integration with the IMU ASAP...

Reply:
> Sure.. I was kinda rushing because you mentioned you wanted to test integration with the IMU ASAP...

No problem. Since I've got to review the PR anyways, I'm comenting :). I can use the driver as-is on the branch for now (not started, have to help with BOM/electrical layout)